configuration/scripts/options/set_nml.tx1

```diff
@@ -3,6 +3,7 @@ runtype = 'initial'
 ice_ic  = 'default'
 grid_format = 'bin'
 grid_type   = 'tripole'
+ns_boundary_type = 'tripole'
```

Comment:
There probably should be a check during the initialization that if grid_type = 'tripole', then ns_boundary_type = 'tripole'.   I haven't heard of any east-west tripole grids, and I don't think CICE's boundary conditions could handle that anyhow.

Reply:
That's a good idea, I'll add it.